apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -473,7 +473,7 @@ public function merge_template_and_post( $post ) {
 		}
 
 		$wrapped_post_content = sprintf( '<!-- wp:a8c/post-content -->%s<!-- /wp:a8c/post-content -->', $post->post_content );
-		$post->post_content   = str_replace( '<!-- wp:a8c/post-content /-->', $wrapped_post_content, $template->get_template_content() );
+		$post->post_content   = str_replace( '<!-- wp:a8c/post-content {"align":"full"} /-->', $wrapped_post_content, $template->get_template_content() );
```

Comment:
Can we make this a bit more robust? We should have a server side block parser available from Gutenberg.

If the above is too expensive/difficult alternatively I'd recommend doing a partial match on `wp:a8c/post-content`, then matching the full html comment.